libs/platform/src/lib/form/multi-combobox/commons/base-multi-combobox.ts

```diff
@@ -685,7 +685,7 @@ export abstract class BaseMultiCombobox extends CollectionBaseInput implements O
 
                 this._suggestions = this._convertToOptionItems(data).map((optionItem: SelectableOptionItem) => {
                     const selectedElement = this._selectedSuggestions.find(
-                        (selectedItem: SelectableOptionItem) => selectedItem.label === optionItem.label
+                        (selectedItem: SelectableOptionItem) => selectedItem.value === optionItem.value
```

Comment:
In terms of `.value`, we should probably make something different, if object, then those values might not match. You have some selectedItems, and then you receive a new on from that stream and references will be different. we cannot rely it can be the same. 

So we need some uniquely identifiable value. or we do more deep object comparision